osu.Game.Rulesets.Catch/Skinning/CatchSkinConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Catch.Skinning
+{
+    public enum CatchSkinConfiguration
```

Comment:
shouldn't this be `CatchSkinColour`?

Reply:
I thought the `XXXSkinColour` enum concept is for doing a lookup on the custom colours of the skin configuration, but no harm in renaming here.

Reply:
Isn't that what you're doing? I'm confused.

Reply:
as in `SkinConfiguration.CustomColours` which is retrieved from the `Colours` section.